### ISSUE TITLE
Log

### ISSUE THREAD

**Issue by alemori** [cncjs/cncjs#743]
#### Description

Is there any possibility to raise a log of the executed commands ?

#### Versions

- CNCjs: 1.9.22

**Comment by emcniece** [comment]
Yes, you can get more log output with `cncjs -vvv`. How are you running CNCjs?
